charts/payment-integration/README.md

```diff
@@ -0,0 +1,96 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
+
+- [Payment Integration Service Helm Chart](#payment-integration-service-helm-chart)
+  - [Chart Details](#chart-details)
+    - [Verification](#verification)
+    - [Installation](#installation)
+    - [Service specific mandatory environment variables](#service-specific-mandatory-environment-variables)
+    - [Example](#example)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Payment Integration Service Helm Chart
+
+This is the Helm Chart for:
+  - [_commercetools Payone Integration Service_](https://github.com/commercetools/commercetools-payone-integration)
+  - [_commercetools PayPal Plus Integration Service_](https://github.com/commercetools/commercetools-paypal-plus-integration) 
+service applications
+
+
+* Kubernetes 1.8+ with beta APIs enabled
+
+## Chart Details
+
+This chart will do the following:
+
+* Deploy an instance of the service (_Payone_ or _Paypal_) as on all nodes of a K8s cluster.
+
+### Verification
+
+Lint the chart
+
+```
+$ helm lint
+```
+
+To test the generated k8s resources from the helm chart
+
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
+```
+
+### Installation
+
+To install the chart with the release name `my-payment-integration` using a dedicated namespace(recommended):
+
+```
+$ helm upgrade --install --namespace my-shop-namespace my-payment-integration -f values.yaml .
+```
+
+The chart can be customized using the following configurable parameters:
+
+| Parameter                       | Description                         | Default                             |
+| ------------------------------- | ------------------------------------|-------------------------------------|
+| `image.repository`              | docker image repository             | ***mandatory***                     |
```

Comment:
it would be better to have proper default values for `image.x` params

Reply:
unfortunately, there are no default values - theses services are absolutely different